cms/api_test.py

```diff
@@ -63,29 +76,42 @@ def test_filter_and_sort_catalog_pages():  # pylint:disable=too-many-locals
     )
 
     all_pages, program_pages, course_pages = filter_and_sort_catalog_pages(
-        initial_program_pages, initial_course_pages, external_course_pages
+        initial_program_pages,
+        initial_course_pages,
+        external_course_pages,
+        external_program_pages,
     )
 
     # Combined pages and course pages should not include the past course run
     assert len(all_pages) == (
         len(initial_program_pages)
         + len(initial_course_pages)
         + len(external_course_pages)
+        + len(external_program_pages)
         - 1
```

Comment:
Shouldn't this be `-2`, as there we consider an earlier program as well now? Can a program with a `run start` date in the past be visible to the user?

Reply:
why `-2` ?